pkg/controller/operands/cdi.go

```diff
@@ -104,19 +99,7 @@ func getDefaultFeatureGates() []string {
 	return []string{HonorWaitForFirstConsumerGate}
 }
 
-func (h *cdiHooks) postFound(req *common.HcoRequest, exists runtime.Object) error {
-	err := h.ensureKubeVirtStorageRole(req)
-	if err != nil {
-		return err
-	}
-
-	err = h.ensureKubeVirtStorageRoleBinding(req)
-	if err != nil {
-		return err
-	}
-
-	return nil
-}
+func (h *cdiHooks) postFound(req *common.HcoRequest, exists runtime.Object) error { return nil }
```

Comment:
This is the one and an only real implementation of the `postFound` function. Please remove it from the interface and delete all the empty implementations (and the the function call too).

Reply:
Done